tests/migrations/test_commands.py

```diff
@@ -463,6 +463,15 @@ def test_sqlmigrate_forwards(self):
             "Transaction end not found or found before operation description (unique_together)"
         )
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_db_constraint"})
+    def test_dbconstraint_alter_field_migration(self):
+        out = io.StringIO()
+        call_command("sqlmigrate", "migrations", "0002", stdout=out)
```

Comment:
use single quoted strings everywhere - will be in code style guide soon as per #8284